lib/message.ex

```diff
@@ -309,11 +310,23 @@ defmodule LangChain.Message do
       do: true
 
   def is_function_call?(%Message{}), do: false
+
+  def add_image(%Message{role: :user, images: images} = message, image) do
```

Comment:
Adding docs would help. It appears from the tests that this is passing a URL? Not the actual image data, correct?

Reply:
Yeah, in this case it is, I'm pretty sure there's an option here to pass base64, so a pattern match on a the image (is_file) or somesuch could handle it easily enough. :+1: on the docs though.